scripts/read_csv_file.py

```diff
@@ -0,0 +1,32 @@
+import csv
+
+def read_csv_file(path):
+    with open(path, mode='r') as csv_file:
+        csv_reader = csv.reader(csv_file)
+        colnames = next(csv_reader)
+        content = []
+        for lines in csv_reader:
+            content.append(lines)
+    return colnames, content
+
+def display_columns(col_choice, max_rows, content):
+    for lines in content[0:max_rows]:
+        print(lines[col_choice-1])
+
+
+
+
```

Comment:
Too much whitespace in between, hehe! flake8 will be angry

Reply:
Hi I have removed the extra spaces.